chanfitness/chanevent.go

```diff
@@ -60,14 +60,18 @@ type chanEventLog struct {
 	closedAt time.Time
 }
 
-func newEventLog(outpoint wire.OutPoint, peer route.Vertex,
+// newEventLog creates an event log for a channel with the openedAt time set.
+func newEventLog(channelPoint wire.OutPoint, peer route.Vertex,
 	now func() time.Time) *chanEventLog {
 
-	return &chanEventLog{
-		channelPoint: outpoint,
+	eventlog := &chanEventLog{
+		channelPoint: channelPoint,
 		peer:         peer,
 		now:          now,
+		openedAt:     now(),
 	}
+
+	return eventlog
```

Comment:
What happened to setting the initial event here with the same timestamp? It is still described in the commit message.

Reply:
I've decided to remove it, rather have channels that exist at startup have a small gap between monitored time and online time than have the bug prone code path (it would also sometimes result in 2x consecutive online events).